### ISSUE TITLE
bug: watch mode not updating when `.graphql` file is changed

### ISSUE THREAD

**Issue by elijaholmos** [anvilco/spectaql#546]
I have set up a simple project with the following file structure:
```
- package.json
- pnpm-lock.yaml
- schema.graphql
- spectaql.yml
```

The `spectaql.yml` config looks like so:
```yml
introspection:
    schemaFile: ./schema.graphql

servers:
    - url: https://example.com/graphql
      description: Test
```

When I run the command `pnpm exec spectaql -D spectaql.yml`, Spectaql successfully generates documentation and creates a development server at http://localhost:4400, then enters watch mode. However, when I save the `schema.graphql` file, the changes are not reflected in neither the website nor the CLI. But when I save the `spectaql.yml` file, the changes are registered in the CLI and the output
```cmd
>> File "spectaql.yml" changed.
```
is displayed. I double-checked the documentation to ensure there was nothing my configuration. Let me know if there are any other steps I should try.

SpectaQL version: 1.5.6
OS: Windows 10 21H2
Node version: 16.18.0
PNPM version: 7.14.1

**Comment by newhouse** [comment]
This is fixed in `v1.5.7`

**Comment by elijaholmos** [comment]
thank you!
